frontend/public/components/overview/_overview.scss

```diff
@@ -3,6 +3,7 @@
   flex-direction: column;
   flex-wrap: wrap;
   justify-content: space-between;
+  min-height: 113px;
```

Comment:
Maybe a comment explaining the significance?

Reply:
comment added